README.md

```diff
@@ -899,6 +899,16 @@ This is an x86_64 machine
 
 - `env_var(key)` — Retrieves the environment variable with name `key`, aborting if it is not present.
 
+```make
```

Comment:
Code blocks with `make` syntax are extracted and parsed, to make sure that they're correct syntax, so the test is failing when it reaches `/home/user`. This should be wrapped in a separate code block of type `sh`.

Reply:
Thanks! Fixed.